app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,7 @@
   <%= javascript_include_tag "application" %>
   <%= csrf_meta_tags %>
   <%= yield :extra_headers %>
+  <%= yield :extra_head_content %>
```

Comment:
I don't think we need this; `content_for` will collect all input given and yielding it outputs them all, so where we've got `content_for(:extra_head_content)` we can re-use `content_for(:extra_headers)` and it should be fine.  FWIW: I thing `extra_head_content` is a better name, so if you wanted to change `extra_headers` to `extra_head_content` I'd be into that.